Model/Cron.php

```diff
@@ -531,6 +536,10 @@ protected function _declareVariables($notification)
             if ($acquirerReference != "") {
                 $this->_acquirerReference = $acquirerReference;
             }
+            $ratepayDescriptor = isset($additionalData['openinvoicedata.descriptor']) ? $additionalData['openinvoicedata.descriptor'] : null;
```

Comment:
```suggestion
            $ratepayDescriptor = isset($additionalData['openinvoicedata.descriptor']) ? $additionalData['openinvoicedata.descriptor'] : "";
```

if you do this you can perform a strict check in the next line for the empty string - maybe it's a better approach